client/v1/src/main/java/org/opennms/velocloud/client/v1/VelocloudApiClientV1.java

```diff
@@ -146,13 +148,54 @@ public List<Edge> getEdges(final String enterpriseId) throws VelocloudApiExcepti
         try {
             final var edges = this.enterpriseApi.enterpriseGetEnterpriseEdges(new EnterpriseGetEnterpriseEdges());
             return edges.stream()
-                    .map(e -> Edge.builder()
-                            .withEnterpriseId(enterpriseId)
-                            .withSite(e.getSite().getName())
-                            .withOperator(e.getConfiguration().getOperator().getName())
-                            .withHub(e.isIsHub())
-                            .build())
-                    .collect(Collectors.toList());
+                        .map(e -> Edge.builder()
+                                      .withEnterpriseId(enterpriseId)
```

Comment:
This is somewhat bogus. A tenant connection always has some kind of intrinsic enterprise ID. But blindly setting it from the parameter is wrong. TBH, the whole way we handle the enterprise ID in tenant connections is wrong.

Reply:
My understanding is that when the customer/tenant credentials are stored, the enterpriseId of their system is saved with them. When we get a request with an alias to those customer's credentials, we get that enterpriseId from SCV to use. Individual requests should never be able to specify the enterpriseId, just the alias of the credentials to use.